src/DataStructures/Tensor/Python/TensorData.cpp

```diff
@@ -10,6 +9,7 @@
 #include <utility>
 
 #include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/TensorData.hpp"
```

Comment:
This is the source file's corresponding header. Don't move it down here, but make it the very first `#include`, followed by a blank line:
```cpp
#include "DataStructures/Tensor/TensorData.hpp"

#include <memory>
// ...
```

Reply:
I looked at other .cpp bindings files and they use the same convention of including the hpp file with other file includes and not as the first include.

Reply:
You’re right, confused this with regular source/header pairs. Just ignore my comment.